### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by andrewbanchich**
Add crates.io and docs badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+[![Documentation](https://docs.rs/onig/badge.svg)](
+https://docs.rs/onig)
 ![CI](https://github.com/rust-onig/rust-onig/workflows/CI/badge.svg)
 [![Build status](https://ci.appveyor.com/api/projects/status/7qxdb44xpw4bkjfi/branch/master?svg=true)](https://ci.appveyor.com/project/iwillspeak/rust-onig/branch/master)
 [![dependency status](https://deps.rs/crate/onig/6.1.0/status.svg)](https://deps.rs/crate/onig/6.1.0)
 
 Rust bindings for the [Oniguruma regex library][Onig_wiki], a powerful and mature regular expression library with support for a wide range of character sets and language syntaxes. Oniguruma is written in C. This repository provides two crates: `onig-sys` which provides the raw Rust FFI bindings, and `onig`, which provides a safe Rust wrapper around them.
 
-## Documentation
+## Examples
```
Comment (iwillspeak) [inline README.md:13]: Can you revert this change please. I know there’ll be two links to the docs but I don’t think that’s a bad thing.

**Review by iwillspeak — CHANGES_REQUESTED** [review]
Looks good, but can you just add the badge please.

File: README.md (hunk 1 above)
Comment (andrewbanchich) [reply README.md]: Sure! All set.

**Review by iwillspeak — APPROVED** [review]
Awesome!
